src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -84,7 +86,15 @@ private static boolean isValidPath(Path filePath) {
      * @throws StorageOperationException if there were errors converting and/or storing data to file.
      */
     public void save(AddressBook addressBook) throws StorageOperationException {
-
+    	
+    	/**
+    	 * Checking for file existing.
+    	 * Throw exception if fail to locate existing file.
+    	 */
+    	if(!fileIsExisting() && fileIsExisted){
+    		throw new StorageOperationException("File not found: " + path);
```

Comment:
You need to keep throwing the exception until it reach the main method. Also, consider creating the file and showing the problem to the user, instead of closing the program.